tests/test_translator.py

```diff
@@ -277,6 +281,232 @@ def test_parse_raw_request_no_path(self):
         with self.assertRaises(ValueError):
             translator.parse_raw_request(raw_request)
 
+    def test_process_arguments_with_no_arguments(self):
+        # Arguments
+        args = argparse.Namespace(
```

Comment:
I'm saying, a function like ``namespace_loader(**kwargs)`` to which you provide something like 
data="sample data", request="GET  HTTP/1.1\nHost: www.google.com".
so, the namespace loader returns you an object with the kwargs provided. rest everything can be set to a default value, which is ``None`` here.